prov/efa/src/rdm/rxr_ep.c

```diff
@@ -1656,16 +1687,16 @@ static inline void rdm_ep_poll_ibv_cq_ex(struct rxr_ep *ep, size_t cqe_to_proces
 			break;
 		}
 
-		pkt_entry = (void *)(uintptr_t)ep->ibv_cq_ex->wr_id;
```

Comment:
why remove this. For RECV_RDMA_WITH_IMM, this will just be NULL. Right?

Reply:
I moved it below to the relevant case statements so that we wouldn't accidentally try to de-reference it in future revision.